client/my-sites/posts/main.jsx

```diff
@@ -50,3 +101,20 @@ module.exports = React.createClass( {
 	}
 
 } );
+
+export default connect( ( state ) => {
+	const selectedSite = getSelectedSite( state );
+	const siteId = selectedSite && selectedSite.ID;
+	const draftsQuery = {
+		type: 'post',
+		status: 'draft',
+		order_by: 'modified'
+	};
+
+	return {
+		drafts: getSitePostsForQueryIgnoringPage( state, siteId, draftsQuery ),
```

Comment:
Good candidate for using the `getSitePostsForQuery` selector instead, since you only care about a single page of data. Whether you change it, highlights that we probably want normalization on that selector as well.